src/steamship/agents/schema/agent.py

```diff
@@ -20,6 +21,9 @@ class Agent(BaseModel, ABC):
     tools: List[Tool]
     """Tools that can be used by the Agent in selecting the next Action."""
 
+    conversation_memory: ConversationMemory = NoMemory()
+    """Conversation memory to use when running agent."""
```

Comment:
btw I had no idea this was a way to document variables in python and ever since seeing you do it have adopted it -- thanks!!